supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -241,3 +445,63 @@ The practice of having both modes together is often referred to as "hybrid", "ag
 *Incorrect forms*: builtin, built in
 
 *See also*:
+
+// EAP: Added "In Red Hat JBoss Enterprise Application Platform," and removed "in JBoss EAP" later
+[discrete]
+[[built-in-messaging]]
+==== image:images/yes.png[yes] built-in messaging (noun)
+*Description*: In Red Hat JBoss Enterprise Application Platform, "built-in messaging" is an acceptable term for referring to the built-in messaging system. Capitalize "built-in" only at the beginning of a sentence. Other acceptable terms are "JBoss EAP messaging" and "JBoss EAP built-in messaging".
+
+*Use it*: yes
+
+*Incorrect forms*: ActiveMQ, ActiveMQ Artemis
+
+*See also*: xref:jboss-eap-built-in-messaging[JBoss EAP built-in messaging], xref:jboss-eap-messaging[JBoss EAP messaging]
+
+// BxMS: Added "In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite," and removed from later in the sentence
+[discrete]
+[[business-central]]
+==== image:images/yes.png[yes] Business Central (noun)
+*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, the "Business Central" is a web-based user interface. It is the user interface for the business rules manager and has been combined with the core Drools engine and other tools. It enables a business user to manage rules in a multi-user environment and implement changes in a controlled fashion.
+
+*Use it*: yes
+
+*Incorrect forms*: Central, BC
+
+*See also*:
+
+// BxMS: General; kept as is
+[discrete]
+[[business-process]]
+==== image:images/yes.png[yes] business process (noun)
+*Description*: A "business process" is a collection of related, structured tasks that results in achieving a specific target. It is presented as as a flowchart comprising a sequence steps necessary to achieve business goals.
```

Comment:
```suggestion
*Description*: A "business process" is a collection of related, structured tasks that results in achieving a specific target. It is presented as a flowchart comprising a sequence of steps necessary to achieve business goals.
```

Reply:
I would avoid the word "comprising". You have used it correctly, but most people do not. How about:

"It is presented as a flowchart that consists of a sequence of steps"